src/libpriv/rpmostree-postprocess.c

```diff
@@ -2144,7 +2145,12 @@ rpmostree_commit (int            rootfs_fd,
         goto out;
     }
   
-  if (refname)
+  if (refname && write_commitid_to)
```

Comment:
Although I could probably just leave setting the parent revision up to if people want it.
If you prefer, I can make this not require refname if write_commitid_to is provided.